tests/conftest.py

```diff
@@ -10,6 +10,7 @@
 import pytest
 
 import glass
+from glass.cosmology import Cosmology
```

Comment:
Are `glass.cosmology.Cosmology` and `cosmology` (type checking block and `_check_version`) both required here? Can we then refer to `glass.cosmology.Cosmology` as `glass.cosmology.Cosmology` in the code to avoid confusion?

Reply:
This is a lazy merge conflict...